Formula/u/unitycatalog.rb

```diff
@@ -0,0 +1,44 @@
+class Unitycatalog < Formula
+  desc "Open, Multi-modal Catalog for Data & AI"
+  homepage "https://unitycatalog.io/"
+  url "https://github.com/unitycatalog/unitycatalog/archive/refs/tags/v0.3.0.tar.gz"
+  sha256 "fae708a22f1e38e19f754aca22925d66016a7efeaab680ce87c27496c75078d1"
+  license "Apache-2.0"
+
+  depends_on "sbt" => :build
+  depends_on "openjdk@21"
+
+  def install
+    system "sbt", "createTarball"
```

Comment:
Honestly, this feels weird to create `jar` file. `tar` and unpack, get `jar`..
Are there any more natural way to create jar artifact?